tough-ssm/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.1] - 2020-07-20
+### Added
+- Added CI build of `tough-ssm`.
+
+### Changed
+- Update `tough` dependency to 0.8.0.
```

Comment:
I'm fairly certain that tough_ssm should be 0.3.0 because the traits implemented are part of tough, which has a "breaking change" bump. If I run `cargo update` and tough stays at 0.7.1 but tough_ssm moves to this new version, they can no longer interoperate because the traits are from different versions of the crate.

Reply:
Good catch - thanks!